src/pybind/mgr/telemetry/module.py

```diff
@@ -1058,6 +1287,64 @@ def send(self,
             return 1, '', '\n'.join(success + failed)
         return 0, '', '\n'.join(success)
 
+    def format_perf_histogram(self, report: Dict[str, Any]) -> None:
+        # Formatting the perf histograms so they are human-readable. This will change the
+        # ranges and values, which are currently in list form, into strings so that
+        # they are displayed horizontally instead of vertically.
+        try:
+            # Formatting ranges and values in osd_perf_histograms
+            modes_to_be_formatted = ['osd_perf_histograms_aggregated', 'osd_perf_histograms_separated']
```

Comment:
I noticed a small thing. The only mode for histograms that exists now is separated, and that was shortened to "osd_perf_histograms". The diff on the left shows this change, but it must have happened in a rebase after this PR was opened.

```suggestion
            modes_to_be_formatted = ['osd_perf_histograms']
```

Reply:
thanks for noticing, @ljflores, changed to use the current (separated) formatting: https://github.com/ceph/ceph/pull/44251/commits/2a2ea49aa370a3ac756c9b5a60b9bb323a6fb808